flow/record/adapter/avro.py

```diff
@@ -117,13 +118,13 @@ def __iter__(self):
             if not self.selector or self.selector.match(rec):
                 yield rec
 
-    def close(self):
+    def close(self) -> None:
         if self.fp:
             self.fp.close()
         self.fp = None
 
 
-def descriptor_to_schema(desc):
+def descriptor_to_schema(desc: record.RecordDescriptor) -> Dict[str, Any]:
```

Comment:
```suggestion
def descriptor_to_schema(desc: record.RecordDescriptor) -> dict[str, Any]:
```
This should be legal if you add a `from __future__ import annotations` at the top.